test/testdata/java_shelf_book_name.baseline

```diff
@@ -27,19 +27,19 @@ import java.util.List;
 public class ShelfBookName extends BookName {
 
   private static final PathTemplate PATH_TEMPLATE =
-      PathTemplate.createWithoutUrlEncoding("shelves/{shelf_id}/books/{book_id}");
+      PathTemplate.createWithoutUrlEncoding("shelves/{shelf_id}/books/{return}");
```

Comment:
In this file, all forms of `book_id` are replaced with the corresponding form for `return`.